.nvmrc

```diff
@@ -1 +1 @@
-v9.8.0
+v10.0.0
```

Comment:
`v9.8.0` is the node version that the current electron version uses. They should stay in sync.

Reply:
Makes sense.